src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int CAPACITY = 10;
+    private final K[] keyArray;
+    private final V[] valueArray;
+    private int size = 0;
```

Comment:
no need to initialize it with 0, primitives have default values in class fields
```suggestion
    private int size;
```